opencraft/settings.py

```diff
@@ -49,6 +49,8 @@
 
 DEBUG = env.bool('DEBUG', default=False)
 
+ENABLE_DEBUG_TOOLBAR = False
```

Comment:
This should be

    ENABLE_DEBUG_TOOLBAR = env.bool('ENABLE_DEBUG_TOOLBAR', default=False)

so we can change it via the `.env` or the environment if needed.